evergreen/evergreen.yml

```diff
@@ -357,6 +470,354 @@ tasks:
+      commands:
+        - func: run-valid-ocsp-server-delegate-responder
+          vars:
+            OCSP_ALGORITHM: "rsa"
+        - func: ocsp-bootstrap-mongo-orchestration
+          vars:
+            ORCHESTRATION_FILE: "rsa-basic-tls-ocsp-mustStaple.json"
+        - func: run-ocsp-test
+          vars:
+            OCSP_ALGORITHM: "rsa"
+            OCSP_TLS_SHOULD_SUCCEED: "true"
+
+    - name: test-ocsp-rsa-invalid-cert-server-staples-delegate-responder
+      tags: ["ocsp"]
+      commands:
+        - func: run-revoked-ocsp-server-delegate-responder
+          vars:
+            OCSP_ALGORITHM: "rsa"
+        - func: ocsp-bootstrap-mongo-orchestration
+          vars:
+            ORCHESTRATION_FILE: "rsa-basic-tls-ocsp-mustStaple.json"
+        - func: run-ocsp-test
+          vars:
+            OCSP_ALGORITHM: "rsa"
+            OCSP_TLS_SHOULD_SUCCEED: "false"
+
+    - name: test-ocsp-rsa-valid-cert-server-does-not-staple-delegate-responder
+      tags: ["ocsp"]
+      commands:
+        - func: run-valid-ocsp-server-delegate-responder
+          vars:
+            OCSP_ALGORITHM: "rsa"
+        - func: ocsp-bootstrap-mongo-orchestration
+          vars:
+            ORCHESTRATION_FILE: "windows/rsa-basic-tls-ocsp-disableStapling.json"
+        - func: run-ocsp-test
+          vars:
+            OCSP_ALGORITHM: "rsa"
+            OCSP_TLS_SHOULD_SUCCEED: "true"
+
+    - name: test-ocsp-rsa-invalid-cert-server-does-not-staple-delegate-responder
+      tags: ["ocsp"]
+      commands:
+        - func: run-revoked-ocsp-server-delegate-responder
+          vars:
+            OCSP_ALGORITHM: "rsa"
+        - func: ocsp-bootstrap-mongo-orchestration
+          vars:
+            ORCHESTRATION_FILE: "windows/rsa-basic-tls-ocsp-disableStapling.json"
+        - func: run-ocsp-test
+          vars:
+            OCSP_ALGORITHM: "rsa"
+            OCSP_TLS_SHOULD_SUCCEED: "false"
+
+    - name: test-ocsp-rsa-malicious-invalid-cert-mustStaple-server-does-not-staple-delegate-responder
+      tags: ["ocsp"]
+      commands:
+        - func: run-revoked-ocsp-server-delegate-responder
+          vars:
+            OCSP_ALGORITHM: "rsa"
+        - func: ocsp-bootstrap-mongo-orchestration
+          vars:
+            ORCHESTRATION_FILE: "windows/rsa-basic-tls-ocsp-mustStaple-disableStapling.json"
+        - func: run-ocsp-test
+          vars:
+            OCSP_ALGORITHM: "rsa"
+            OCSP_TLS_SHOULD_SUCCEED: "false"
+
+    - name: test-ocsp-rsa-malicious-no-responder-mustStaple-server-does-not-staple-delegate-responder
+      tags: ["ocsp"]
+      commands:
+        - func: ocsp-bootstrap-mongo-orchestration
+          vars:
+            ORCHESTRATION_FILE: "windows/rsa-basic-tls-ocsp-mustStaple-disableStapling.json"
+        - func: run-ocsp-test
+          vars:
+            OCSP_ALGORITHM: "rsa"
+            OCSP_TLS_SHOULD_SUCCEED: "false"
+
+    # ECDSA tests
+    # Disabled until https://jira.mongodb.org/browse/SPEC-1589 is resolved
```

Comment:
These configs are valid but result in system failures due to mongod's inability to read ecdsa certs as pem files on Windows. See Spec ticket for more information